src/App.vue

```diff
@@ -257,12 +291,13 @@ export default {
             title: '',
         },
 
-        importExportModal: {
+        importModal: {
             data: '',
-            title: '',
         },
 
-        isImport: false,
+        exportModal: {
+            data: '',
+        },
```

Comment:
If you are using just one variable, you don't need the object, use directly the exportModal

Reply:
Same for the importModal